packages/radio/test/radio-group.test.ts

```diff
@@ -629,3 +635,29 @@ describe('Radio Group', () => {
         expect(changeSpy.calledWith(undefined)).to.be.false;
     });
 });
+
+describe('Radio Group - late children', () => {
+    it.only('accepts frame late children', async () => {
+        const test = await fixture(html`
```

Comment:
We should also note why this is a required test.
```suggestion
        /**
         * In some cases (e.g. when wrapped in React components) will cause otherwise standard looking
         * DOM structures to add `<sp-radio>` children to `<sp-radio-group>` parents in a non-syncronous manner.
         * 
         * This test emulates that render process to ensure that validation will still work as expect in that context.
         */
        const test = await fixture(html`
```